[Documentation] Update Network Manager documentation and Wiki known issues. (#2586)
Azure/ResourceModules: docs/wiki/Known issues.md

## Microsoft.Network/networkManagers

> In order to deploy network manager where the `networkManagerScopes` property is set to `managementGroups`, you need to register the management group with the `Microsoft.Network` resource provider first, which is available in the [Providers - Register At Management Group Scope API](https://learn.microsoft.com/en-us/rest/api/resources/providers/register-at-management-group-scope) documentation.

Reviewer: ```suggestion
In order to deploy a Network Manager with the `networkManagerScopes` property set to `managementGroups`, you need to register the `Microsoft.Network` resource provider at the Management Group first ([ref](https://learn.microsoft.com/en-us/rest/api/resources/providers/register-at-management-group-scope)).
```
